src/factory.js

```diff
@@ -29,6 +30,25 @@ function getObjectMemberExpression (name) {
   }
 }
 
+function getNestedObjectMemberExpression (node) {
```

Comment:
Prawdopodobnie nie ogra to bardziej skomplikowanych przypadków, typu `foo[bar[baz]]`, ale na razie jest ok